src/modules/nodejs.rs

```diff
@@ -92,8 +92,8 @@ fn get_engines_version(context: &Context) -> Option<String> {
     Some(raw_version.to_string())
 }
 
-fn check_engines_version(nodejs_version: &str, engines_version: Option<String>) -> bool {
-    if engines_version.is_none() {
+fn check_engines_version(nodejs_version: Option<&String>, engines_version: Option<String>) -> bool {
+    if engines_version.is_none() || nodejs_version.is_none() {
         return true;
     }
```

Comment:
I would prefer not calling `unwrap` unless strictly necessary to avoid issue in case of potential refactorings, Perhaps use something like this?

```suggestion
    let (nodejs_version, engines_version) = match (nodejs_version, engines_version) {
        (Some(nv), Some(ev)) => (nv, ev),
        _ => return true,
    };
```